lib/cursor.js

```diff
@@ -102,11 +110,8 @@ class Cursor {
     query // filter expression
     // Return: new Cursor instance
   ) {
-    const cursor = new Cursor.MemoryCursor();
-    cursor.parents.push(this);
-    if (this.schema) cursor.definition(this.schema);
-    cursor.jsql.push({ op: 'select', query });
-    return cursor;
+    this.jsql.push({ op: 'select', query });
```

Comment:
I'd prefer `select` to generate new cursor

Reply:
I don't understand the reasoning behind generating a new Cursor, could you please elaborate?
As I see it, `select` on a Cursor is just a select operation of a data which doesn't imply the creation of a new cursor.

Reply:
Create new name for `select`.
/cc @tshemsedinov